lib/videoroom_web/templates/room/index.html.heex

```diff
@@ -22,13 +22,17 @@
                     <audio></audio>
                     <video class="w-full h-full"></video>
                     <div name="video-label" class="absolute text-white text-shadow-lg bottom-0 left-0 p-2">Placeholder</div>
+                    <img class="invisible invert" src={static_path(@conn, "/svg/mic-off-fill.svg")} style="position: absolute; left: 0; top: 0; width: 10%, height: 10%">
+                    <div name="no-video-msg" class="invisible absolute text-white w-full h-full top-0 left-0" style="text-align: center">No video feed</div>
```

Comment:
Could we make order of `<div> <img> <div>` the same as for the `remote-video-feed-template` or this was done on purpose?